### DISCUSSION TITLE
How to add a new dag in docker

### DISCUSSION THREAD

**Discussion by HugoPu** [apache/airflow#14657] [Q&A]
I ran airflow in docker,  add a new dag python file to the `dags` folder which is set in the [`docker-compose.yaml`](https://airflow.apache.org/docs/apache-airflow/stable/start/docker.html)， refreshed the `8080`website, and can't find any thing is changed, how can I add a new dag in docker?

**Comment by kaxil** [comment]
Where was your `dags` folder relative to the `docker-compose.yaml`?
